packages/bigquery/src/table.js

```diff
@@ -644,6 +579,10 @@ Table.prototype.createWriteStream = function(metadata) {
       metadata: {
         configuration: {
           load: metadata
+        },
+        jobReference: {
+          jobId: jobId,
+          projectId: self.bigQuery.projectId
```

Comment:
I don't believe we test that `projectId` is in the request.

Reply:
https://github.com/callmehiphop/gcloud-node/blob/79bcb4d0b6ea0c8ce2b42f83cd95a99b87cb1951/packages/bigquery/test/table.js#L655